globus_sdk/__init__.py

```diff
@@ -30,7 +31,8 @@
 
     "GlobusResponse", "GlobusHTTPResponse",
 
-    "GlobusError", "GlobusAPIError", "TransferAPIError", "SearchAPIError",
+    "GlobusError", "GlobusSDKUsageError",
```

Comment:
Do you need to introduce this new error class in `docs/exceptions.rst` as well, including maybe rephrasing the "malformed calls to Globus SDK methods may raise standard python exceptions" bit?

Reply:
Yeah, I think a doc update would be appropriate. Will add.